### DISCUSSION TITLE
Add build information to pom.xml

### DISCUSSION THREAD

**Discussion by tgodzik** [sbt/sbt#7901] [Ideas]
It was raised in the Scala 3 repository that it would be useful to add some information about how the code was compiled to published jars. More context here https://github.com/scala/scala3/issues/21894

I wonder how much of an issue would it be to add such information to pom.xml? Or is it even doable?

The alternative would be to have some properties file added to jars, but that too would need to be solved on the build tool level.

Such properties could be:
- scala version
- scala options
- plugins with coordinates (that's probably another cause to put it into pom.xml)

Opinions? Maybe someone can point us to the place to take a look at?

**Comment by eed3si9n** [comment]
`versionScheme` info could be considered a prior art on this type of information, probably - https://github.com/sbt/sbt/pull/5724
